index.js

```diff
@@ -27,4 +29,32 @@ form.addEventListener('submit', (e) => {
     e.preventDefault();
     errorMsg.classList.add('display-content');
   }
+});
+
+// Enable Local storage for the form
+
+function populateStorage() {
+  const nameValue = userName.value;
+  const emailValue = email.value;
+  const userMessage = messageContent.value;
+
+  const userInputs = {
+    name: nameValue,
+    email: emailValue,
+    message: userMessage,
+  };
+
+  localStorage.setItem(userInputs, JSON.stringify(userInputs));
+}
+
+const parsedInputs = JSON.parse(localStorage.getItem('userInputs'));
```

Comment:
-  First, you should check if there is data in the local storage. 
-  If there is data then you should get that data from local storage and fill the form fields with that data
-  If there is no data previously then you should store data in local storage
-  With this method, you'll not lose data from the user when the page loads it will check the storage and update the UI.